client/app/dedicated/server/bandwidth/private-order/private-order.controller.js

```diff
@@ -40,50 +43,49 @@ class ServerOrderBandwidthVrackCtrl {
         load: () => {
           this.isLoading = true;
           return this.Server
-            .getBareMetalPrivateBandwidthOrder(this.$stateParams.productId, this.model.plan)
+            .getBareMetalPrivateBandwidthOrder(this.serverName, this.model.plan)
             .then((res) => {
               res.bandwidth = _.find(this.plans, 'planCode', this.model.plan).bandwidth;
               res.planCode = this.model.plan;
               this.provisionalPlan = res;
             })
             .catch((error) => {
-              this.$scope.resetAction();
-              return this.$scope.setMessage(this.$translate.instant('server_order_bandwidth_vrack_loading_error'), error.data);
+              this.goBack();
```

Comment:
Please chain your promises :)

Reply:
You want to chain a goBack ?

Reply:
Well it's a promise isn't it ? Don't you want to make sure your page is loaded before setting the message ?

Reply:
I'll move that in the finally